components/References/References.tsx

```diff
@@ -0,0 +1,32 @@
+'use client'
+
+// Tailwind Merge
+import { twMerge } from 'tailwind-merge'
+
+// React components
+import SectionTitle from '../SectionTitle'
+import ReferenceItems from '../ReferenceItems'
+
+// Next intl
+import { useTranslations } from 'next-intl'
+
+type ReferencesProps = {
+  className?: string
+}
+
+const References = ({ className }: ReferencesProps) => {
+  const t = useTranslations('Colaboration')
+
+  return (
+    <section
+      className={twMerge('mx-auto max-w-[1408] px-7 lg:px-12', className)}
```

Comment:
nechybí tu u max-w jednotka?

Reply:
kopíroval jsem to z CollaboratingSubjects. Také tam chybí koukám. Přidáno i tam